components/developer/golang-118/golang.p5m

```diff
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright 2021 Till Wegmueller
+# Copyright 2022 Till Wegmueller
 #
 
 set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
```

Comment:
This still seems odd. Adding a new package should only have added lines and no changes or deletions. This looks like git considered this have has been moved from golang-117.

Reply:
Well it has been.